examples/05_glm_second_level/plot_second_level_two_sample_test.py

```diff
@@ -60,26 +60,30 @@
 second_level_input = sample_vertical["cmaps"] + sample_horizontal["cmaps"]
 
 # %%
-# Next, we model the effect of conditions (sample 1 vs sample 2).
+# Next, we assign the subjects for vertical and horizontal checkerboard respectively.
 import numpy as np
 
-condition_effect = np.hstack(([1] * n_subjects, [-1] * n_subjects))
+vertical_subjects = np.hstack(([1] * n_subjects, [0] * n_subjects))
+horizontal_subjects = np.hstack(([0] * n_subjects, [1] * n_subjects))
```

Comment:
Sorry I should have been clearer, but when you do this you make the design matrix  rank deficient: the sum of these two regressors, is equal to the some of all the subject regressors. 
This means that the design matrix is no longer invertible, and some contrasts are not estimable. Normally, you should get a warning when you run that code.